packages/ocamlformat-lib/ocamlformat-lib.0.25.0/ocamlformat-lib.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/vendor/parser-recovery/lib/dune b/vendor/parser-recovery/lib/dune
```

Comment:
Sorry about the lack of reporting (I'll try to fix that in opam-repo-ci). The reason behind it is that this file is not in the right place. It should be placed in the `files/` directory
```
2023-03-01 09:04.29: Job failed: Unexpected path "packages/ocamlformat-lib/ocamlformat-lib.0.25.0/ocamlformat-lib.patch" in output (expecting 'packages/name/pkg/...')
```